BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Loop_Parts/Part_D/Part_D1_Array.sh

```diff
@@ -78,7 +86,7 @@ END
 	cd $WorkingDir
 	numJobs=$((NPOP*Seeds))
 	maxJobs=252 # for now, maybe make this a variable in the main loop script
-	sbatch --array=1-${numJobs}%${maxJobs} --export=ALL,gen=$gen,WorkingDir=$WorkingDir,RunName=$RunName,Seeds=$Seeds,AraSimDir=$AraSimExec --job-name=${RunName} Batch_Jobs/AraSimCall_Array.sh 
```

Comment:
Here we have numJobs = NPOP*Seeds and in the batch job script we move ${num}_${seed}.txt to the AraSimFlags directory. Do we need to change how we're counting/checking flags since we're stacking multiple AraSim runs into each job? Seems like maybe we would have numJobs = NPOP*Seeds/40 if we're running 40 instances of AraSim per job.